### DISCUSSION TITLE
Can I close my account?

### DISCUSSION THREAD

**Discussion by lxu97y** [community/community#68962] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Github team,

I own the account @sephirothxlx. But I can't login this account anymore because I don't have 2FA and recovery code. 
I recently I received an email from my professor that I have some codes posted in my account related to class assignments and I need to take them down. But since I don't have access to my account, I can't do that. 
Could you help me close that account or set it to private? So no one can search the codes/repos from my account? I can provide the information to prove my identity. 
Looking forward to hearing from you! Thanks!

**Comment by radar-blip** [comment]
I think if you contact support, they can disable it for you.
Here is a link for that: [Support page](https://support.github.com/contact/account)

**Reply by lxu97y** [reply]
Thanks!

**Reply by radar-blip** [reply]
Anytime

**Comment by salimovrashad** [comment]
Response #2

**Comment by edglz** [comment]
You can, but you need to contact [support](https://support.github.com/) team of github for disable your account.
